test/offer_schema.js

```diff
@@ -155,7 +157,6 @@ describe("# Offer Schema tests", () => {
                 test("Inside the range should not throw error", async () => {
                     const submitted_fields = [];
                     for (let i = 0; i < MIN_FIELDS; ++i) {
-                        // Preventing interference from duplicate error
```

Comment:
I didn't mean to remove these, only the "returning promise" ones :/
These actually make sense